src/isWilsonPrime.js

```diff
@@ -20,6 +20,17 @@
  */
 function isWilsonPrime(num) {
   // write code here
+    let factNum = num - 1;
```

Comment:
you can check for 0 and 1 in a variable.
```suggestion
   let factNum = num < 2 ? 1 : num - 1;
```
now you no need `if`